spotinst/credential_aws/consts.go

```diff
@@ -0,0 +1,8 @@
+package credential_aws
+
+import "github.com/spotinst/terraform-provider-spotinst/spotinst/commons"
+
+const (
+	IamRole   commons.FieldName = "iamrole"
+	AccountId commons.FieldName = "accountid"
```

Comment:
use camelcase "accountId"

Reply:
camelCase is not allowed , modified name to account_id.